tests/settings/ConfigurationGatheringTest.py

```diff
@@ -83,13 +83,13 @@ def test_gather_configuration(self):
 
         self.assertEqual(len(local_bears['cli']), 0)
 
-    def test_default_coafile_parsing(self):
+    def test_system_coafile_parsing(self):
```

Comment:
you need to test that the old file name still works and outputs a deprecated message.